core/app/modules/import-export/module.php

```diff
@@ -122,82 +122,31 @@ private function render_temp_import_export_content() {
 				],
 				'description' => __( 'Apply the design and settings of another site to this one.', 'elementor' ),
 				'link' => [
-					'url' => '#',
+					'url' => 'https://go.elementor.com/wp-dash-import-export-import-flow',
 					'text' => __( 'Learn More', 'elementor' ),
 				],
 			],
 		];
 
-		$info_text_link = sprintf( '<a href="#">%s</a>', __( 'Click here.', 'elementor' ) );
+		$front_page_id = get_option( 'page_on_front' );
```

Comment:
What should happen if there is no page on front?

Reply:
As agreed with @DorShahar, the info link was removed.